commands/deploy_file.js

```diff
@@ -37,50 +36,34 @@ exports.builder = {
 }
 
 exports.handler = async function (argv) {
-    const content = fs.readFileSync(argv.file)
-    
-    transaction = null
-    txBuilder = archethic.newTransactionBuilder("hosting")
-        .setContent(content)
-    
+
+    setcontent(argv.file)
```

Comment:
It's not clear what it's doing.
What is the output of this function, etc...
I would suggest to use more functional programming approach with the concept of immutable data, so you only return new object when you mutate state.
Then it will be easier to read and understand